### ISSUE TITLE
pwa:create:screenshot dumps an invalid configuration

### ISSUE THREAD

**Issue by tacman** [Spomky-Labs/phpwa#111]
### Version(s) affected

1.1.x

### Description

Using the YAML generated from the screenshot command throws an error

![image](https://github.com/Spomky-Labs/pwa-bundle/assets/619585/bd7d3005-9eda-4794-a42f-eef98d93f2a6)

### How to reproduce

```bash
 bin/console pwa:create:screenshot https://127.0.0.1:8013/ --output=homepage

PWA - Take a screenshot
=======================

                                                                                                                        
 [OK] Screenshot saved. You can now use it in your application configuration file.                                      
                                                                                                                        

pwa:
  screenshots:
    -
      src: homepage/screenshot-1200x1100.png
      width: 1200
      height: 1100
      type: image/png
```

Now move the generated yaml to pwa.yaml and refresh the page

### Possible Solution

Probably the screenshot command yaml dumper hasn't been updated to the latest screenshot config.

**Comment by Spomky** [comment]
This should be fixed now

**Comment by Spomky** [comment]
The result is now as follows:

```yaml
pwa:
  manifest:
    screenshots:
      -
        src: homepage/screenshot-1200x1100.png
        width: 1200
        height: 1100
        format: image/png
```

**Comment by Spomky** [comment]
Closing as fixed in 1.1.0
